src/main/menus/app.js

```diff
@@ -8,242 +8,242 @@ import {app, Menu, session, shell, webContents} from 'electron';
+        role: 'zoomOut',
+    }, separatorItem, {
+        label: 'Developer Tools for Application Wrapper',
+        accelerator: (() => {
+            if (process.platform === 'darwin') {
+                return 'Alt+Command+I';
+            }
+            return 'Ctrl+Shift+I';
+        })(),
+        click(item, focusedWindow) {
+            if (focusedWindow) {
+                // toggledevtools opens it in the last known position, so sometimes it goes below the browserview
+                if (focusedWindow.isDevToolsOpened()) {
+                    focusedWindow.closeDevTools();
+                } else {
+                    focusedWindow.openDevTools({mode: 'detach'});
+                }
+            }
+        },
     }, {
-      role: 'close',
-      accelerator: 'CmdOrCtrl+W',
-    }, separatorItem, ...teams.slice(0, 9).sort((teamA, teamB) => teamA.order - teamB.order).map((team, i) => {
-      return {
-        label: team.name,
-        accelerator: `CmdOrCtrl+${i + 1}`,
+        label: 'Developer Tools for Current Server',
         click() {
-          WindowManager.switchServer(team.name, true);
+            WindowManager.openBrowserViewDevTools();
         },
-      };
-    }), separatorItem, {
-      label: 'Select Next Server',
-      accelerator: 'Ctrl+Tab',
-      click() {
-        WindowManager.sendToRenderer('select-next-tab');
-      },
-      enabled: (teams.length > 1),
-    }, {
-      label: 'Select Previous Server',
-      accelerator: 'Ctrl+Shift+Tab',
-      click() {
-        WindowManager.sendToRenderer('select-previous-tab');
-      },
-      enabled: (teams.length > 1),
-    }],
-  };
-  template.push(windowMenu);
-  const submenu = [];
-  if (config.helpLink) {
-    submenu.push({
-      label: 'Learn More...',
-      click() {
-        shell.openExternal(config.helpLink);
-      },
+    }];
+
+    if (process.platform !== 'darwin') {
+        viewSubMenu.push(separatorItem);
+        viewSubMenu.push({
+            label: 'Toggle Dark Mode',
+            click() {
+                // TODO: review what to do with this one
+                WindowManager.sendToRenderer('set-dark-mode');
+            },
+        });
+    }
+
+    template.push({
+        label: '&View',
+        submenu: viewSubMenu,
+    });
+    template.push({
+        label: '&History',
+        submenu: [{
+            label: 'Back',
+            accelerator: process.platform === 'darwin' ? 'Cmd+[' : 'Alt+Left',
+            click: () => {
+                const focused = webContents.getFocusedWebContents();
+                if (focused.canGoBack()) {
+                    focused.goBack();
+                }
+            },
+        }, {
+            label: 'Forward',
+            accelerator: process.platform === 'darwin' ? 'Cmd+]' : 'Alt+Right',
+            click: () => {
+                const focused = webContents.getFocusedWebContents();
+                if (focused.canGoForward()) {
+                    focused.goForward();
```

Comment:
I don't think there's really a difference between what we do here and what happens in that logic. I think we just use `goToOffset` in the `windowManager` because we want to be able to handle back and forward with one event.

0/5 of whether we change this or not.